tests/components/griddy/test_config_flow.py

```diff
@@ -0,0 +1,52 @@
+"""Test the Griddy Power config flow."""
+from asynctest import MagicMock, patch
+
+from homeassistant import config_entries, setup
+from homeassistant.components.griddy.config_flow import CannotConnect
+from homeassistant.components.griddy.const import DOMAIN
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "griddypower.async_api.AsyncGriddy.async_getnow", return_value=MagicMock(),
```

Comment:
Is the patch applied in the correct place?

https://docs.python.org/3/library/unittest.mock.html#where-to-patch

Reply:
@MartinHjelmare There really isn't any other place to patch since this is the only api call

Reply:
Usually it's safer to patch where the name is used not where it's defined, ie this is usually what's best `"homeassistant.components.griddy.config_flow.AsyncGriddy.async_getnow"`.

Reply:
That is certainly more targeted.  done and pushed